infra/ansible/playbook.yaml

```diff
@@ -21,6 +21,8 @@
           pattern: ^(aarch64|amd64)$
         - name: accelerator
           pattern: ^(tpu|cuda)$
+        - name: disable_xrt
+          pattern: ^(0|1)$
```

Comment:
This shouldn't be here. Only variables that don't have default values and need to be passed on every invocation of `ansible-playbook` should be here. 

The build error is there, because task "validate required variables" runs before "Include vars from config files" (which loads vars from vars.yaml).

The variable can still be overwritten

Reply:
Thanks. I removed the validation here since this flag is temporary and the var gets picked up as expected: https://github.com/pytorch/xla/runs/14112366156